docs/references/nextjs/auth.mdx

```diff
@@ -32,54 +65,72 @@ import { auth } from '@clerk/nextjs';
 
 export async function GET() {
   const {userId, getToken} = auth();
+
   if(!userId){
     return new Response("Unauthorized", { status: 401 });
   }
+
   const token = await getToken({template: "supabase"});
-  // Fetch data from Supabase and return it.
+
+  // Add logic here to fetch data from Supabase and return it.
+  
   const data = { supabaseData: 'Hello World' };
+  
   return NextResponse.json({ data });
 }
 ```
 
-## Authorization
+## Use `auth()` to check if a user is authorized
 
-Use `has` in order to check if a user is authorized to access a component.
+`auth()` returns the `Authentication` object, which includes the `has()` helper. `auth()` also returns the `protect()` helper. 
```

Comment:
```suggestion
`auth()` returns the [`Authentication`](/docs/references/nextjs/authentication-object#authentication-object) object, which includes the `has()` helper. `auth()` also returns the `protect()` helper. 
```

Just for discussion's sake, I know in the style guide it says we should de-dupe reference links, but that should probably be scoped within headings.

I think someone might get a link to this heading, see the `Authentication` object, and want to just be able to click a link to see what that is without having to scroll to the top of the page

Reply:
I like this idea!! I was the one duping reference links within docs, because I do believe it's helpful in the reading journey to not have to scroll back to a first instance if you want to get to the reference. So I can get on board with this :) let's update the styleguide accordingly